cmd/minikube/cmd/tunnel.go

```diff
@@ -103,6 +106,13 @@ var tunnelCmd = &cobra.Command{
 	},
 }
 
+func outputTunnelStarted() {
+	out.Styled(style.Success, "Tunnel successfully started")
+	out.Ln("")
+	out.Styled(style.Notice, "NOTE: This process must stay alive for the tunnel to be accessible ...")
```

Comment:
How about adding 

"Please do not close this terminal, as this process has to stay alive for ....(rest)"

Reply:
Updated!